### ISSUE TITLE
The value of loss is very large

### ISSUE THREAD

**Issue by gravity-lhg** [atomistic-machine-learning/schnetpack#336]
hey，
I implemented multi-attribute training according to the method you said，it worked. 
Although the loss obtained is declining, the value is very large。。。
Some of the following data：
Train loss,                                    Validation loss,
438577113.14285713,                 258545200.8,
113668425.6,                               92633047.86666666,
59218379.89285714,                   43999447.2,
......
Is it caused by the data not being normalized？ if yes. Is there an interface for data normalization？ And how to operate it？

**Comment by Stefaanhess** [comment]
Hey,
you can get the statistics of the dataset with `train_loader.get_statistics(...)`. From that you get the mean and std values, which can be given as arguments to the output-modules.
